### DISCUSSION TITLE
Impulse response files for Convolution reverb?

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#625] [Help]
Originally posted 2020-08-25 23:29:46.296000 by mikhram in GrandOrgue discussion forum on SourceForge<br><br>Hello,

I&#039;d like to use the Convolution reverb feature.

However, the only &quot;approved&quot; response files at
http://familjenpalo.se/vpo/ir-recordings/ are no longer downloadable.

I downloaded the &quot;Echo Thief&quot; library, but most files there cause
GrandOrgue to make random noises. A few files do work but it&#039;s a long
trial and error process.

Could people please recommend free (or cheap) impulse response WAV
files known to work with GO?

-- 
Yours, Mikhail Ramendik

Unless explicitly stated, all opinions in my mail are my own and do
not reflect the views of any organization

**Comment by larspalo** [comment]
Originally posted 2020-08-26 19:37:16.159000 by mikhram in GrandOrgue discussion forum on SourceForge<br><br>In the absence of answers, here is how I got an apparently workable reverb:

- Downloaded Big Gee&#039;s Lexicon 480L collection from here: https://www.housecallfm.com/download-gns-personal-lexicon-480l
- Unzipped
- In the &quot;Big Gee&#039;s Lexicon 480L&quot; subdirectory converted all AIF files to WAV and then downmixed to mono (using ffmpeg - commands available on request)
- Tried stuff :) and settled on &quot;Large Church&quot; with a gain of 0.05. 

The &quot;Large Church&quot; file is *a lot* of reverb. And this enables the reverb to be heard even at a very small gain. The less gain I used, the less distortion of the tone happened, according to the feelings of the organist (who is also my wife). So this combination of &quot;much reverb low gain&quot; gave moderate workable reverb without significant distortion.

**Comment by ahall41** [comment]
For anyone interested, there are also some useful IR files on the [OpenAIR](https://www.openair.hosted.york.ac.uk/) website. One thing I have noticed is that quite a few of them end up with significant resonation at certain frequencies - presumably because the building resonates at that frequency. 

And now that GO IR is starting to become usable ...

**Reply by larspalo** [reply]
Yes, we have a link to IR data on that website in our wiki - https://github.com/GrandOrgue/grandorgue/wiki/Impulse-Responses.
